apps/root/src/pages/earn/components/all-strategies-table/index.tsx

```diff
@@ -0,0 +1,211 @@
+        <FormattedMessage description="allVaultsTVL" defaultMessage="TVL" />
+      </StyledBodySmallLabelTypography>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallLabelTypography>
+        <FormattedMessage description="allVaultsAPY" defaultMessage="APY" />
+      </StyledBodySmallLabelTypography>
+    </TableCell>
+  </TableRow>
+);
+
+const ROWS_PER_PAGE = 7;
+
+const skeletonRows = Array.from(Array(ROWS_PER_PAGE).keys());
+
+const AllStrategiesTableBodySkeleton = () => (
+  <>
+    {skeletonRows.map((i) => (
+      <TableRow key={i}>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <ContainerBox gap={2} alignItems="center">
+            <Skeleton variant="circular" width={28} height={28} animation="wave" />
+            <StyledBodySmallSemiboldTypo2>
+              <Skeleton variant="text" animation="wave" width="6ch" />
+            </StyledBodySmallSemiboldTypo2>
+          </ContainerBox>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <ContainerBox>
+            <StyledYieldTypeBox>
+              <StyledBodySmallRegularTypo2>
+                <Skeleton variant="text" animation="wave" width="6ch" />
+              </StyledBodySmallRegularTypo2>
+            </StyledYieldTypeBox>
+          </ContainerBox>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+      </TableRow>
+    ))}
+  </>
+);
+
+const createRow = (strategy: Strategy) => (
+  <TableRow>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>{strategy.farm.name}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
+      <ContainerBox gap={2} alignItems="center">
+        {strategy.asset.icon}
+        <StyledBodySmallSemiboldTypo2>{strategy.asset.symbol}</StyledBodySmallSemiboldTypo2>
+      </ContainerBox>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>{strategy.network.name}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
+      <ContainerBox>
+        <StyledYieldTypeBox>
+          <StyledBodySmallRegularTypo2>{strategy.farm.yieldType}</StyledBodySmallRegularTypo2>
+        </StyledYieldTypeBox>
+      </ContainerBox>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>${usdFormatter(strategy.farm.tvl)}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
```

Comment:
what about security?

Reply:
Wasn't sure if implement it already, but I can include it now 😄